src/core/inlines.js

```diff
@@ -294,38 +294,36 @@ export function run(conf) {
       matched = !matched;
       if (!matched) {
         df.append(t);
-      } else if (t.startsWith("{{")) {
-        const node = inlineXrefMatches(t, txt);
-        df.append(node);
-      } else if (t.startsWith("[[[")) {
-        const node = inlineRefMatches(t);
-        df.append(node);
-      } else if (t.startsWith("[[")) {
-        const nodes = inlineBibrefMatches(t, txt, conf);
-        df.append(...nodes);
-      } else if (t.startsWith("|")) {
-        const node = inlineVariableMatches(t);
-        df.append(node);
-      } else if (t.startsWith("[=")) {
-        const node = inlineAnchorMatches(t);
-        df.append(node);
-      } else if (t.startsWith("`")) {
-        const node = inlineCodeMatches(t);
-        df.append(node);
-      } else if (t.startsWith("[^")) {
-        const node = inlineElementMatches(t);
-        df.append(node);
-      } else if (abbrMap.has(t)) {
-        const node = inlineAbbrMatches(t, txt, abbrMap);
-        df.append(node);
-      } else if (keywords.test(t)) {
-        const node = inlineRFC2119Matches(t);
-        df.append(node);
-      } else {
-        // FAIL -- not sure that this can really happen
-        throw new Error(
-          `Found token '${t}' but it does not correspond to anything`
-        );
```

Comment:
Might be useful to keep this..

Reply:
As we've never hit this in the last N years, I figured we are safe to just remove it. Something has to match so in *theory* this won't ever trigger... I'm feeling confident in removing it.